cli/flags/common.go

```diff
@@ -24,6 +25,11 @@ const (
 	FlagTLSVerify = "tlsverify"
 )
 
+const (
+	// FlagTimeout is timeout flag
+	FlagTimeout = "timeout"
```

Comment:
Err can we put this with the others in the `const` block above?

Reply:
done